frontend/src/App.vue

```diff
@@ -1,9 +1,44 @@
 <template>
-  <nav>
-    <router-link to="/">Home</router-link> |
-    <router-link to="/about">About</router-link>
-  </nav>
-  <router-view />
+  <div id="content">
+    <header>
+      <v-img
+        :src="require('./assets/drg_logo.png')"
+        class="my-3"
+        contain
+        height="200"
+      />
+    </header>
+    <nav class="navbar navbar-dark bg-secondary border">
+      <router-link style="color: #e0ff2f" class="nav-item nav-link" to="/"
```

Comment:
Also same for color, using 'secondary' will help us have a uniform webapp

Reply:
Sounds good!